src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -78,6 +78,13 @@ module internal Utils =
     module Html =
         let sepWith s l = l |> List.sepWith (!! s) |> span []
 
+    type System.Xml.Linq.XElement with
+        member x.TryAttr (attr: string) =
```

Comment:
I added this for convenience and then only used it in the new code, but I'd of course be willing to reuse it in the existing attribute fetches in this file if you like.